README.md

```diff
@@ -117,6 +118,8 @@ You can use the same steps that in **How to add a new package** just set the var
 - DISTRO_EOX_AUDIT_MODEL_DPKG
 - DISTRO_EOX_TAGGING_DPKG
 
+:warning: **NOTE**: From Olmo version Distro has not defaulted packages. Now it is necessary to add the packages you want in ``config.yml`` file. See [How to add a new package](./docs/how_to_add_new_packages.rst)
```

Comment:
The default packages section doesn't have sense with his changes. I suggest dropping that section. What do you think?
The same with the override and disable packages.

Reply:
You are right on this point, but I do not consider it prudent to remove this content, because we have many users using the Distro with other versions of Tutor. It is better to give some time for users to start using the Distro for this new purpose and we might update the documentation in the future.